great-idea/css/index.css

```
@@ -1,66 +1,359 @@
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* header {
+	position: fixed;
+	z-index: 10;
+} */
+
+header {
+  display: flex;
+}
+
+.nav__container {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 80%;
+}
+.container {
+  display: flex;
+  /*border: 1px solid red;*/
+  width: 80%;
+  justify-content: space-between;
+}
+
+.container li a {
+  display: flex;
+  justify-content: space-around;
+  /*border: 1px solid red;*/
+  text-decoration: none;
+  color: gray;
+}
+
+.logo {
+  align-self: flex-end;
+  justify-self: center;
+}
+
+.hero__container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  width: auto;
+  margin-top: 100px;
+}
+
+.hero__heading {
+  font-size: 80px;
+  text-align: center;
+}
+
+.line {
+  border-top: 1px solid black;
+  margin: 40px;
+}
+
+.btn {
+  border: 2px solid black;
+  width: 200px;
+  text-align: center;
+  box-shadow: 5px 3px 3px black;
+}
+
+.features__container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 20px;
+  text-align: left;
+  margin: 0 70px;
+  /*border: 1px solid red;*/
+}
+
+.features {
+	margin: 0 20px;
+}
+.about {
+	margin: 0 20px;
+}
+.services {
+	margin: 0 20px;
+}
+.product {
+	margin: 0 20px;
+}
+.vision {
+	margin: 0 20px;
+}
+
```

you can do all this in less line of code with something like this.
```suggestion
.features, .about, .services, .ect {
     margin: 0 20px;
}
```